qelem/lib/screens/login_screen.dart

```diff
@@ -96,7 +99,11 @@ class _LoginScreenState extends State<LoginScreen> {
               height: 30.0,
             ),
             ElevatedButton(
-              onPressed: () {},
+              onPressed: () {
+                _authRepo.login(
+                    username: usernameController.text,
+                    password: passwordController.text);
+              },
```

Comment:
??

Reply:
i had to call repo directly because there aren't any viewmodels, forgot to delete.